app/js/lib/foxbox/foxbox.js

```diff
@@ -57,14 +57,7 @@ export default class Foxbox extends Service {
     // polling and triggering box-online events with a boolean
     // indicating if we have access to box or not.
     this._initDiscovery()
-      .then(() => this[p.net].init())
```

Comment:
I've moved service list polling management to view itself, so that it can manage it as it wants. Still there is one edge case - when we're in the Service view - currently we don't poll anything here and I was planning to leave it as is for this PR since we don't have any rich Service view where we need this feature.